### DISCUSSION TITLE
How do i use runpod for sdxl?

### DISCUSSION THREAD

**Discussion by Ibxx0403** [TheLastBen/fast-stable-diffusion#2876] [Q&A] [answered]
As pps doesn't work anymore how do i use runpod, i simply cannot find a template from the last ben or any SDXL lora template. Is there something i am doing wrong or am i missing something. Thanks in advance

**Comment by TheLastBen** [comment, marked as answer]
In RunPod, the template name is "fast-stable-diffusion" https://www.runpod.io/console/deploy?template=runpod-stable-unified

**Reply by Ibxx0403** [reply]
Thanks i was looking for "the last ben", SDXL and Lora/Dreambooth and couldn't find anything. Also what would be the "recommended" gpu be for best price to performance?, and for information i use a pretty large image set (up to 1000 images).

**Reply by TheLastBen** [reply]
a 3090 would be fine, also, use runpod community servers, they are cheaper

**Reply by Ibxx0403** [reply]
Thanks i will try it out today

**Reply by Ibxx0403** [reply]
i got it to work!!! 

<img width="772" alt="Screenshot 2024-07-08 at 10 03 02" src="https://github.com/TheLastBen/fast-stable-diffusion/assets/148635617/f1ea5a93-e70e-4ddd-b26b-9bab5147f076">

**Reply by TheLastBen** [reply]
great!
